src/main/java/strategy/SupplyStrategy.java

```diff
@@ -0,0 +1,19 @@
+package strategy;
+
+import db.Dao;
+import db.DaoImpl;
+import service.Strategy;
+
+public class SupplyStrategy implements Strategy {
```

Comment:
давай називати ці класи SupplyHandler
клас Strategy теж слід назвати більш інформативно, уявімо що в майбутньому тобі потрібно буде додати ще одну стратегію

Reply:
Interestingly, in my previous, version I had "handlers". But, the common mistakes file says that we should have package called "strategy". So, to be consistent with naming I renamed all relevant classes using strategy instead of handler. But I agree, we should consider the possibility that we will have another strategy in the future. So, now to meet both requirements I create a new package called "handler" inside the strategy package, call my classes "...handler" again and put them inside that package.